src/client/react.tsx

```diff
@@ -39,10 +39,12 @@ export type BoardProps<G extends any = any> = ClientState<G> &
     isMultiplayer: boolean;
   };
 
+export type CtxClient = Omit<Ctx, 'random' | '_random'>
```

Comment:
There’s a missing `;` here. Try running `npm run prettier` on your changes to fix any style errors like this.